arch/risc-v/src/mpfs/mpfs_irq.c

```diff
@@ -41,7 +41,7 @@
  * Public Data
  ****************************************************************************/
 
-volatile uint64_t *g_current_regs[1];
+volatile uint64_t *g_current_regs;
```

Comment:
I think this should be reverted to keep the same style with other files.
```suggestion
volatile uintptr_t *g_current_regs[1];
```